.github/.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+# 📄 PR Description
+
+[Add a description for this pull request. Follow the checklist below.] 
+<!--- Describe your changes in detail including a relevant motivation and context for this change -->
+
+
+# ✒️ Checkliste PR-Author
+
+## Before submitting the PR, please make sure you do the following
+
+### Prepare Code for PR
+* [ ] The automated tests were executed locally (`mvn clean test`), and it was ensured that they passed.
+* [ ] All commented-out code and debug statements have been removed.
+* [ ] A test is included that fails without this PR but passes with it.
+* [ ] The relevant documentation, including code comments and public documentation, has been updated.
+    * [ ] [Qytera Github-Page (github.io)](https://github.com/Qytera-Gmbh/Qytera-Gmbh.github.io)
+
+
+### PR Description
+* [ ] A message body is added that provides context and guidance so that the reviewer can quickly understand what the PR does. In the PR body, this includes. In the PR body, includes:
+    * [ ] The purpose of the pull request, clearly illustrating what problem this PR solves and why this solution is appropriate.
+    * [ ] An overview of the changes, ideally with 1-2 examples that illustrate the changes.
+    * [ ] Links to any additional context such as tracking issues or tickets.
+
+### Review Preparation
+* [ ] Review from appropriate team members is requested.
+- Be prepared to explain and discuss changes.
```

Comment:
```suggestion
```

I think this is unnecessary. It doesn't add any value (you wouldn't create a PR if you weren't ready for feedback).